nowcast/delphi_nowcast/data_containers.py

```diff
@@ -0,0 +1,95 @@
+"""Data container classes for holding sensor configurations and data needed for fusion."""
+
+from dataclasses import dataclass, field
+from typing import List, Union
+
+from numpy import ndarray, nan, nanmean, isnan
+from pandas import date_range
+
+
+@dataclass(frozen=True)
+class SensorConfig:
```

Comment:
Are we using this anywhere?

Reply:
oops not in this PR, sorry for including it. There's some examples in this draft: https://github.com/cmu-delphi/covidcast-indicators/pull/745/files#diff-75869a055fa7d473f31362ae54a8733cafe3e693333f2b9377e0d60d727eba23R115